plugins/modules/discovery_intent.py

```diff
@@ -119,16 +119,16 @@
       snmp_priv_protocol:
         description: SNMP privacy protocol (DES/AES128)
         type: str
-      snmp_ro_community:
+      snmp_r_o_community:
```

Comment:
In SNMP, we have snmp read-only community, hence we can write as snmp_ro_community and it will be more meaningful.